src/domain/useCase/updateService.go

```diff
@@ -15,7 +15,11 @@ func UpdateService(
 	mappingCmdRepo repository.MappingCmdRepo,
 	updateDto dto.UpdateService,
 ) error {
-	serviceEntity, err := servicesQueryRepo.ReadByName(updateDto.Name)
+	readInstalledDto := dto.ReadInstalledServicesItemsRequest{
+		Name:                 &updateDto.Name,
+		ShouldIncludeMetrics: false,
+	}
+	serviceEntity, err := servicesQueryRepo.ReadUniqueInstalledItem(readInstalledDto)
```

Comment:
Why the Unique prefix?

Reply:
I discuss this in more detail in the second request change below.